packages/sources/paypal/README.md

```diff
@@ -0,0 +1,232 @@
+# Chainlink External Adapter for Paypal
+
+External adapter connecting to the [Paypal payouts API](https://developer.paypal.com/docs/api/payments.payouts-batch/v1/) using the [Paypal payout SDK](https://github.com/paypal/Payouts-NodeJS-SDK)
+
+**Disclaimer**  
+In order to use this adapter, you will need to create an account with and obtain credentials from PayPal and agree to and comply with PayPal’s applicable terms, conditions and policies. In no event will SmartContract Chainlink Limited SEZC be liable for your or your user’s failure to comply with any or all of PayPal’s terms, conditions or policies or any other applicable license terms.
+
+### Environment Variables
+
+| Required? |      Name       |                                       Description                                        |      Options      | Defaults to |
+| :-------: | :-------------: | :--------------------------------------------------------------------------------------: | :---------------: | :---------: |
+|    ✅     |   `CLIENT_ID`   | [API access credential](https://developer.paypal.com/docs/api/overview/#get-credentials) |                   |             |
+|    ✅     | `CLIENT_SECRET` | [API access credential](https://developer.paypal.com/docs/api/overview/#get-credentials) |                   |             |
+|           |     `MODE`      |                 Parameter for determining a live or sandbox environment                  | `sandbox`, `live` |  `sandbox`  |
```

Comment:
Maybe this should be an input param rather than an env var. If I was a user seems like the flow would always be to try sandbox and then switch to live.

Reply:
Hmmmm interesting suggestions. Here's what I was thinking:
* I like it defaulting to `sandbox` because it means the user or EA operator has to make a very clear choice to send a 'live' transaction
* In terms moving the `mode` parameter to an input param, i don't think it would really work to flip between the two environments because the `CLIENT_ID` and `CLIENT_SECRET` are specific to one environment or the other (based on my understanding).

Reply:
Ah, shoot. Alright if that is the case then LGTM.